src/PKSim.Presentation/Presenters/Main/MenuAndToolBarPresenter.cs

```diff
@@ -543,21 +551,23 @@ private void updateParameterIdentificationItems(ParameterIdentification paramete
 
       public void Handle(SensitivityAnalysisStartedEvent sensitivityAnalysisEvent)
       {
-         _sensitivityRunning = true;
+         _runningSensitivityAnalysis = sensitivityAnalysisEvent.SensitivityAnalysis;
          updateSensitivityAnalysisItems(sensitivityAnalysisEvent.SensitivityAnalysis);
       }
 
       public void Handle(SensitivityAnalysisTerminatedEvent sensitivityAnalysisEvent)
       {
-         _sensitivityRunning = false;
+         _runningSensitivityAnalysis = null;
```

Comment:
Sensitivity analyses cannot be run in parallel, only PIs, correct?

Reply:
There is no change on the sensitivityAnalysis logic so far